tests/shopify_python/test_packaging.py

```diff
@@ -0,0 +1,102 @@
+import contextlib
+import re
+import subprocess
+
+from git import repo
+import pytest
+
+from shopify_python import packaging
+
+
+@pytest.fixture
+def package_source_root(tmpdir):
+    # type: ('py.path.LocalPath') -> 'py.path.LocalPath'
+    root = tmpdir.join('test_packaging')
+
+    root.join('setup.py').write('\n'.join([
+        "import setuptools",
+        "setuptools.setup(",
+        "    name='test_packaging',",
+        "    install_requires=[",
+        "        'shopify_python'",
+        "    ],",
+        "    entry_points={",
+        "        'egg_info.writers': [",
+        "            'git_sha.txt = shopify_python.packaging:write_package_revision',",
+        "        ],",
+        "    }",
+        ")"]),
+        ensure=True
+    )
```

Comment:
Without linefeeds it's not valid code. I think it would work if I add a semi-colon after the import line. WDYT?